alphadia/outputtransform.py

```diff
@@ -509,20 +515,17 @@ def build_precursor_table(
 
             if not os.path.exists(psm_path):
                 logger.warning(f"no psm file found for {raw_name}, skipping")
-                run_df = pd.DataFrame()
             else:
                 try:
                     run_df = pd.read_parquet(psm_path)
+                    psm_df_list.append(run_df)
                 except Exception as e:
                     logger.warning(f"Error reading psm file for {raw_name}")
                     logger.warning(e)
-                    run_df = pd.DataFrame()
-
-            psm_df_list.append(run_df)
 
         if len(psm_df_list) == 0:
-            logger.error("No psm files found, can't continue")
-            raise FileNotFoundError("No psm files found, can't continue")
+            logger.error("No psm files accumulated, can't continue")
+            raise OutputGenerationError("No psm files accumulated, can't continue")
```

Comment:
this is a quite generic .. could we make this more specific, e.g. "`NoPsmFound`"? the benefit would be that the we could have fine-granular error codes, which makes alphaDIA automation better to control